code/game/gamemodes/game_mode.dm

```diff
@@ -279,11 +279,18 @@
 			intercepttext += i_text.build(A)
 		else
 			intercepttext += i_text.build(A, pick(modePlayer))
-
-	print_command_report(intercepttext,"Centcom Status Summary")
-	priority_announce("Summary downloaded and printed out at all communications consoles.", "Enemy communication intercept. Security Level Elevated.", 'sound/AI/intercept.ogg')
-	if(security_level < SEC_LEVEL_BLUE)
-		set_security_level(SEC_LEVEL_BLUE)
+	
+	var/list/sec = get_living_sec()
+	for (var/obj/machinery/vending/security/S in machines)
+		if(sec.len < 3)
+			print_command_report(intercepttext,"Centcom Status Summary")
+			priority_announce("Summary downloaded and printed out at all communications consoles.", "Enemy communication intercept. Security level elevated. Critical security staff levels detected, additional supplies will be teleported to the brig.  ", 'sound/AI/intercept.ogg')
+			var/obj/structure/closet/secure_closet/lowsec/C = new /obj/structure/closet/secure_closet/lowsec(S.loc)

```

Comment:
Also you don't need to initialize this as a var unless you're modifying the object.